spec/integration/v2/reports_spec.rb

```diff
@@ -208,8 +208,8 @@
       operationId 'SystemReports'
       content_types
       pagination_params_v2
-      sort_params_v2(V2::Report)
-      search_params_v2(V2::Report)
+      sort_params_v2(V2::Report, except: %i[os_major_version])
+      search_params_v2(V2::Report, except: %i[os_major_version with_reported_systems])
```

Comment:
For some reason I'm able to filter reports under system using these filters
![image](https://github.com/user-attachments/assets/0ba64276-711e-4775-a450-636cea9835c0)